samples/msal-react-samples/react-router-sample/src/App.js

```diff
@@ -12,6 +12,13 @@ import { PageLayout } from "./ui-components/PageLayout";
 import { Home } from "./pages/Home";
 import { Profile } from "./pages/Profile";
 
+/**
+ * Comment the line above and uncomment the line below for a 
+ * Class-based equivalent of "Profile" component wrapped with "withMsal" HOC
+ */
+
+// import { ProfileWithMsal } from "./pages/ProfileClassComponent";
```

Comment:
Why not just have them live side by side? You can add a routes below e.g. `/profileWithMsal` and `profileRawContext`

This would also allow us to write e2e tests against these approaches

Reply:
Good idea -done!